doc/release/1.7.0-notes.rst

```diff
@@ -24,53 +24,290 @@ For running on PyPy, PyPy3 6.0+ is required.
 
+Improved input validation and error messages for ``fitpack.bispev`` and
+``fitpack.parder`` for scenarios that previously caused substantial confusion
+for users.
+
 `scipy.io` improvements
 -----------------------
 
 
 `scipy.linalg` improvements
 ---------------------------
 
+An LAPACK wrapper was added for access to the ``tgexc`` subroutine.
+
 
 `scipy.ndimage` improvements
 ----------------------------
 
+`scipy.ndimage.affine_transform` is now able to infer the ``output_shape`` from
+the ``out`` array.
 
 `scipy.optimize` improvements
 -----------------------------
 
+The optional parameter ``bounds`` was added to 
+`scipy.optimize.optimize._minimize_neldermead` to support bounds constraints
+for the Nelder-Mead solver.
+
+``trustregion`` methods ``trust-krylov``, ``dogleg`` and ``trust-ncg`` can now
+estimate ``hess`` by finite difference using one of 
+``["2-point", "3-point", "cs"]``.
+
+``halton`` was added as a ``sampling_method`` in `scipy.optimize.shgo`. 
+``sobol`` was fixed and is now using `scipy.stats.qmc.Sobol`.
+
+``halton`` and ``sobol`` were added as ``init`` methods in 
+`scipy.optimize.differential_evolution.`
+
+``differential_evolution`` now accepts an ``x0`` parameter to provide an
+initial guess for the minimization.
+
+``least_squares`` has a modest performance improvement when SciPy is built
+with Pythran transpiler enabled
 
 `scipy.signal` improvements
 ---------------------------
 
+``get_window`` supports ``general_cosine`` and ``general_hamming`` window 
+functions.
+
+`scipy.signal.medfilt2d` now releases the GIL where appropriate to enable
+performance gains via multithreaded calculations.
 
 `scipy.sparse` improvements
 ---------------------------
 
+Addition of ``dia_matrix`` sparse matrices is now faster.
+
+
 `scipy.spatial` improvements
 ----------------------------
 
+``distance.cdist`` and ``distance.pdist`` performance has greatly improved for 
+certain weighted metrics. Namely: ``minkowski``, ``euclidean``, ``chebyshev``,
+``canberra``, and ``cityblock``.
+
+Modest performance improvements for many of the unweighted ``cdist`` and 
+``pdist`` metrics noted above.
+
+The parameter ``seed`` was added to `scipy.spatial.vq.kmeans` and 
+`scipy.spatial.vq.kmeans2`.
+
+The parameters ``axis`` and ``keepdims`` where added to 
+`scipy.spatial.distance.jensenshannon.`
+
+The ``rotation`` methods `from_rotvec` and `as_rotvec` now accept a `degrees`
+argument to specify usage of degrees instead of radians.
+
+`scipy.special` improvements
+--------------------------
+
+Wright's generalized Bessel function for positive arguments was added as
+`scipy.special.wright_bessel.`
+
+An implementation of the inverse of the Log CDF of the Normal Distribution is
+now available via `scipy.special.ndtri_exp`.
+
 `scipy.stats` improvements
 --------------------------
 
+An overhaul of the Mann-Whitney-Wilcoxon test, ``mannwhitneyu``, including
+support n-dimensional input, an exact test method, and improved documentation.
```

Comment:
This is overlapping with a `mannwhitneyu` description further down - can you merge the two?

Reply:
I'll take care of that. This one is probably better; I'll delete the other one and reference "Other Changes".